src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -432,26 +435,47 @@ public Uri GetTeamProjectUrl(string projectName, string teamName)
             return builder.Uri;
         }
 
+
+        public async Task HandleLoginAsync(bool showDialog=false, Uri serverUri = null)
```

Comment:
As a rule, it's not a good idea to pass bools into a function unless you are setting a bool value. The problem is that when looking at the function call, the info about what the bool represents may not be present. Even if you rely on intellisense, the comments could easily get out-of-date.

Reply:
Changed to use `CredentialPromptType`, which is what the bool eventually got converted into